pkg/ccl/logictestccl/tests/local/local_test: TestCCLLogic_crdb_internal failed

pkg/ccl/logictestccl/tests/local/local_test.TestCCLLogic_crdb_internal [failed](https://tanzanite.cluster.engflow.com/invocation/9f917b34-099d-49ca-aa75-b4acb6f02fad) on master @ [2859e5fab431536ec77f646787af637eec3d185c](https://github.com/cockroachdb/cockroach/commits/2859e5fab431536ec77f646787af637eec3d185c):


```
=== RUN   TestCCLLogic_crdb_internal
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestCCLLogic_crdb_internal1024344097
    test_log_scope.go:81: use -show-logs to present logs inline
[09:37:10] setting distsql_workmem='24372B';
    logic.go:4186: -- test log scope end --
test logs left over in: outputs.zip/logTestCCLLogic_crdb_internal1024344097
--- FAIL: TestCCLLogic_crdb_internal (46.72s)
=== RUN   TestCCLLogic_crdb_internal/node_tenant_capabilities_cache
    logic.go:2908: 
         to prevent flakes in queries that return multiple rows, add the rowsort option, the valuesort option, the partialsort option, or an ORDER BY clause. If you are certain that your test will not flake due to a non-deterministic ordering of rows, you can add the nosort option to ignore this error
[09:37:56] --- progress: /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/crdb_internal: 22 statements
    logic.go:2177: 
         /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/crdb_internal:198: too many errors encountered, skipping the rest of the input
[09:37:56] --- done: /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/crdb_internal with config local: 22 tests, 2 failures
[09:37:56] --- total progress: 22 statements
--- total: 22 tests, 2 failures
    --- FAIL: TestCCLLogic_crdb_internal/node_tenant_capabilities_cache (45.21s)
```
<p>Parameters: <code>attempt=1</code>
, <code>run=13</code>
, <code>shard=6</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCCLLogic_crdb_internal.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-33848

pkg/ccl/logictestccl/tests/local/local_test.TestCCLLogic_crdb_internal [failed](https://tanzanite.cluster.engflow.com/invocation/8dd72aed-8fea-46dd-8ce5-44c52fa58893) on master @ [caff15394fcbe37208b46b2973714c27cc3a1417](https://github.com/cockroachdb/cockroach/commits/caff15394fcbe37208b46b2973714c27cc3a1417):


```
=== RUN   TestCCLLogic_crdb_internal
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestCCLLogic_crdb_internal1205139560
    test_log_scope.go:81: use -show-logs to present logs inline
    logic.go:4186: -- test log scope end --
test logs left over in: outputs.zip/logTestCCLLogic_crdb_internal1205139560
--- FAIL: TestCCLLogic_crdb_internal (46.75s)
=== RUN   TestCCLLogic_crdb_internal/node_tenant_capabilities_cache
    logic.go:2908: 
         to prevent flakes in queries that return multiple rows, add the rowsort option, the valuesort option, the partialsort option, or an ORDER BY clause. If you are certain that your test will not flake due to a non-deterministic ordering of rows, you can add the nosort option to ignore this error
[08:49:44] --- progress: /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/crdb_internal: 22 statements
    logic.go:2177: 
         /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/crdb_internal:198: too many errors encountered, skipping the rest of the input
[08:49:44] --- done: /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/crdb_internal with config local: 22 tests, 2 failures
[08:49:44] --- total progress: 22 statements
--- total: 22 tests, 2 failures
    --- FAIL: TestCCLLogic_crdb_internal/node_tenant_capabilities_cache (45.20s)
```
<p>Parameters: <code>attempt=1</code>
, <code>run=12</code>
, <code>shard=6</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCCLLogic_crdb_internal.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/logictestccl/tests/local/local_test.TestCCLLogic_crdb_internal [failed](https://tanzanite.cluster.engflow.com/invocation/858831e6-8b3c-4bae-a5ed-2531f4a68078) on master @ [01e65172dcb17384db33e8229d16461f6f99c01d](https://github.com/cockroachdb/cockroach/commits/01e65172dcb17384db33e8229d16461f6f99c01d):


```
=== RUN   TestCCLLogic_crdb_internal
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestCCLLogic_crdb_internal2242295143
    test_log_scope.go:81: use -show-logs to present logs inline
    logic.go:4186: -- test log scope end --
test logs left over in: outputs.zip/logTestCCLLogic_crdb_internal2242295143
--- FAIL: TestCCLLogic_crdb_internal (46.71s)
=== RUN   TestCCLLogic_crdb_internal/node_tenant_capabilities_cache
    logic.go:2908: 
         to prevent flakes in queries that return multiple rows, add the rowsort option, the valuesort option, the partialsort option, or an ORDER BY clause. If you are certain that your test will not flake due to a non-deterministic ordering of rows, you can add the nosort option to ignore this error
[09:12:08] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/crdb_internal: 22 statements
    logic.go:2177: 
         /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/crdb_internal:198: too many errors encountered, skipping the rest of the input
[09:12:08] --- done: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/crdb_internal with config local: 22 tests, 2 failures
[09:12:08] --- total progress: 22 statements
--- total: 22 tests, 2 failures
    --- FAIL: TestCCLLogic_crdb_internal/node_tenant_capabilities_cache (45.20s)
```
<p>Parameters: <code>attempt=1</code>
, <code>run=3</code>
, <code>shard=6</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCCLLogic_crdb_internal.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/logictestccl/tests/local/local_test.TestCCLLogic_crdb_internal [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsGcpLinuxX8664BigVm_CclUnitTests/12852907?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsGcpLinuxX8664BigVm_CclUnitTests/12852907?buildTab=artifacts#/) on master @ [0b82e156811000e4a0f8147a1e6325352417285a](https://github.com/cockroachdb/cockroach/commits/0b82e156811000e4a0f8147a1e6325352417285a):


```
=== RUN   TestCCLLogic_crdb_internal
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/07c32376813dce8cb782e888fc5e9d97/logTestCCLLogic_crdb_internal1883247714
    test_log_scope.go:81: use -show-logs to present logs inline
[07:22:44] setting distsql_workmem='70318B';
    logic.go:4186: -- test log scope end --
--- FAIL: TestCCLLogic_crdb_internal (51.26s)
=== RUN   TestCCLLogic_crdb_internal/node_tenant_capabilities_cache
    logic.go:2908: 
         to prevent flakes in queries that return multiple rows, add the rowsort option, the valuesort option, the partialsort option, or an ORDER BY clause. If you are certain that your test will not flake due to a non-deterministic ordering of rows, you can add the nosort option to ignore this error
[07:23:31] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/5343/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/crdb_internal: 22 statements
    logic.go:2177: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/5343/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/crdb_internal:198: too many errors encountered, skipping the rest of the input
[07:23:31] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/5343/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/crdb_internal with config local: 22 tests, 2 failures
[07:23:33] --- total progress: 22 statements
--- total: 22 tests, 2 failures
    --- FAIL: TestCCLLogic_crdb_internal/node_tenant_capabilities_cache (45.33s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCCLLogic_crdb_internal.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/logictestccl/tests/local/local_test.TestCCLLogic_crdb_internal [failed](https://tanzanite.cluster.engflow.com/invocations/default/628f4b1e-e8b7-41f6-a501-bf15569cbdae?testReportRun=12&testReportShard=6&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy9sb2NhbDpsb2NhbF90ZXN0) on master @ [1f53a61732c6f9e48622f45d276eb5e1c343ad30](https://github.com/cockroachdb/cockroach/commits/1f53a61732c6f9e48622f45d276eb5e1c343ad30):


```
=== RUN   TestCCLLogic_crdb_internal
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestCCLLogic_crdb_internal947692210
    test_log_scope.go:81: use -show-logs to present logs inline
[07:06:13] setting distsql_workmem='92425B';
    logic.go:4193: -- test log scope end --
test logs left over in: outputs.zip/logTestCCLLogic_crdb_internal947692210
--- FAIL: TestCCLLogic_crdb_internal (46.71s)
=== RUN   TestCCLLogic_crdb_internal/node_tenant_capabilities_cache
    logic.go:2915: 
         to prevent flakes in queries that return multiple rows, add the rowsort option, the valuesort option, the partialsort option, or an ORDER BY clause. If you are certain that your test will not flake due to a non-deterministic ordering of rows, you can add the nosort option to ignore this error
[07:06:59] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/crdb_internal: 22 statements
    logic.go:2184: 
         /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/crdb_internal:198: too many errors encountered, skipping the rest of the input
[07:06:59] --- done: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/crdb_internal with config local: 22 tests, 2 failures
[07:06:59] --- total progress: 22 statements
--- total: 22 tests, 2 failures
    --- FAIL: TestCCLLogic_crdb_internal/node_tenant_capabilities_cache (45.20s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=12</code>
 - <code>shard=6</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCCLLogic_crdb_internal.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/logictestccl/testdata/logic_test/crdb_internal
+++ b/pkg/ccl/logictestccl/testdata/logic_test/crdb_internal
@@ -188,21 +188,22 @@ statement ok
 SELECT crdb_internal.create_tenant(5)
 
 # Use retry because source data is eventually consistent.
-query ITT colnames,retry
-SELECT * FROM crdb_internal.node_tenant_capabilities_cache WHERE capability_name = 'can_admin_split'
+query ITT colnames,retry,rowsort
+SELECT * FROM crdb_internal.node_tenant_capabilities_cache WHERE capability_name = 'can_view_node_info'
 ----
-tenant_id  capability_name    capability_value
-1          can_admin_split  true
+tenant_id  capability_name     capability_value
+1          can_view_node_info  true
+5          can_view_node_info  false
 
 statement ok
-ALTER TENANT [5] GRANT CAPABILITY can_admin_split
+ALTER TENANT [5] GRANT CAPABILITY can_view_node_info
 
 # Use retry because source data is eventually consistent.
 query ITT colnames,retry,rowsort
-SELECT * FROM crdb_internal.node_tenant_capabilities_cache WHERE capability_name = 'can_admin_split'
+SELECT * FROM crdb_internal.node_tenant_capabilities_cache WHERE capability_name = 'can_view_node_info'
 ----
 tenant_id  capability_name  capability_value
-1          can_admin_split  true
-5          can_admin_split  true
+1          can_view_node_info  true
+5          can_view_node_info  true
 
 subtest end

